cmd/wampproto/main_test.go

```diff
@@ -0,0 +1,46 @@
+package main_test
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	main "github.com/xconnio/wampproto-cli/cmd/wampproto"
+)
+
+func TestRunGenerateChallenge(t *testing.T) {
```

Comment:
These tests can be simplified, no need to loop, no need to check regex. If the output is supposed to be hex, just decode and if it works its correct. Same for base64.